### PR TITLE
Fix broken roundtrip by allowing guid's in create requests

### PR DESCRIPTION
**Pull request by jasollien**
Fix broken roundtrip by allowing guid's in create requests

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -765,6 +766,7 @@ JSON encoded body using the "application/json" content type.
 |bim_snippet.reference|string|Reference of a BIM-Snippet of a topic|false|
 |bim_snippet.reference_schema|string|Schema of a BIM-Snippet of a topic|false|
 |due_date|string|Until when the topics issue needs to be resolved|false|
+|guid|string|GUID of the topic. The server may ignore this and instead generate it's own guid|false|
```
Comment (AlainBolduc) [inline README.md]: This is a duplicate of line 753. This should have been put on line 753 instead.

File: Schemas_draft-03/Collaboration/DocumentReference/document_reference_POST.json [hunk 2]
```diff
@@ -2,6 +2,9 @@
     "title": "document_reference_POST",
     "type": "object",
     "properties": {
+        "guid": {
```
Comment (AlainBolduc) [inline Schemas_draft-03/Collaboration/DocumentReference/document_reference_POST.json]: Unless I don't understand the purpose of document reference, this makes no sense to include the "guid" here. Document reference should refer to an _existing_ document in the project.

File: README.md (hunk 1 above)
Comment (jasollien) [reply README.md]: Fix here: https://github.com/buildingSMART/BCF-API/pull/195. Thanks for reporting

File: Schemas_draft-03/Collaboration/DocumentReference/document_reference_POST.json (hunk 2 above)
Comment (jasollien) [reply Schemas_draft-03/Collaboration/DocumentReference/document_reference_POST.json]: A document reference could be either a link to an existing document or to an external url. For external URL's I think it makes sence to include the guid.

**Pull request merged**